### PR TITLE
554 integration tests gh action

### LINKED ISSUE #554: Hook up integration tests to ci pipeline
We should run our integration tests in the pipeline for this repo.

Keep in mind:

smelling that this relies on a Chrome browser being available on the host? Need to test it in Github actions

_Originally posted by @giorgiosironi in https://github.com/libero/reviewer/pull/553_

### PR DESCRIPTION
**Pull request by erkannt**
## Notes:

- github hosted runners come with most dependencies including chrome [see list](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md)
- told nightwatch to run Chromium headless
- added setup_yarn to setup target run by test_integration target
- copied workflow from libero/reviewer-submission

## Issues:

- took out caching of yarn
  - post action fails due to root:root perm on .data folder used by etcd and minio
  - yarn install only takes 7s

## Open questions, unknowns on my part:
- does this break how you use the makefile for local development?
- do repo settings need changing to remove travis ci requirement/enable github action requirement?  
  I don't have permissions for this.

Closes #554

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Need to remove the required Travis CI status check in https://github.com/libero/reviewer/settings/branch_protection_rules/6394978 if everyone's alright with that?

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,7 @@ create_networks:
 setup:
```
Comment (giorgiosironi) [inline Makefile]: shouldn't these three just be dependent targets? Makes them parallelizable, idiomatic, etc

Comment (erkannt) [reply Makefile]: Yes. After discussing with Erez we agreed to move this to a separate PR. I think it something that is also present in other makefiles.

**Comment by erkannt** [comment]
The requirement for a travis.ci pass have been removed.

PRs now require CI/intergration-tests to pass.

**Pull request merged**